### ISSUE TITLE
[PSaaS FR]: NA values for unburned cells for all statistics

### ISSUE THREAD

**Issue by RobBryce** [PSaaS-Developers/Project_issues#75]
### Contact Details

rbryce@heartlandsoftware.ca

### What is needed?

REDApp #1258 changed some outputs for fire arrival time, where outside the fire bounds, 0 was originally used, which was changed to NODATA.

Currently, if you export a grid of any other statistic, such as FI values, 0 is used out the fire bounds.  I'm asking that we can optionally change 0 to NODATA for outside the fire perimeter, for consistency with fire arrival time exports.  I'm asking that this is optional so that there's backwards compatibility and consistency issues with prior datasets.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by spydmobile** [comment]
Can we get some weigh in here @nealmcloughlin and @BadgerOnABike ?

**Comment by nealmcloughlin** [comment]
I support this feature request to be able to assign a NoData value to raster cells outside of a fire boundary for all gridded outputs. This helps differentiate with certainty between a cell statistic of zero and unburned cells outside the fire. It's also better for mapping.

**Comment by BadgerOnABike** [comment]
This makes good sense to me as well. Especially if people are looking to calculate anything across the landscape, 0's will confound those statistics.

**Comment by spydmobile** [comment]
@RobBryce This just needs a plan and we can move ahed on it.

**Comment by RobBryce** [comment]
The coding is pretty straightforward, since it's only adding a little functionality to what we already have.  Validation will occur easily too.  The work has to cover multi-band TIF exports too, though.

We need to incorporate this as an option (if it is to be an option) into the FGMJ/proto definition, both for loading and saving.

**Comment by spydmobile** [comment]
To clarify @RobBryce is the comment above meant to be the plan?

**Comment by RobBryce** [comment]
@spydmobile  Yes, this work is trivial, as has been discussed and noted here.  There are no architectural changes needed.  It's simply adding code to existing functionality and export specification, and performing testing.  At least half of the time will need to be dedicated to testing, to ensure that sub-scenario outputs are generated correctly.

**Comment by spydmobile** [comment]
From Exec: This will be always on (Default behaviour) - not optional, please proceed with quotation if you have not already.
